kitsune/community/templates/community/top_contributors_react.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% set styles = ('common', 'community-new',) %}
+{% if area == 'questions' %}
+    {% set scripts = ('community-new-questions',) %}
+{% elif area == 'l10n' %}
+    {% set scripts = ('community-new-l10n',) %}
+{% endif %}
+
+{% if area == 'questions' %}
+    {% set title = _('Top Contributors - Support Forum') %}
+{% elif area == 'kb' %}
+    {% set title = _('Top Contributors - Knowledge Base') %}
+{% elif area == 'l10n' %}
+    {% set title = _('Top Contributors - Localization') %}
+{% elif area == 'army-of-awesome' %}
+    {% set title = _('Top Contributors - Army of Awesome') %}
+{% endif %}
+{% set crumbs = [(url('community.home'), _('Community Hub')),
+                 (None, title)] %}
+
+{% block content %}
+    <script type="application/json" name="contributor-data">{{ contributors_json|safe }}</script>
+    <script type="applicatoin/json" name="locale-data">{{ locales_json|safe }}</script>
```

Comment:
I thought we avoided inline scripts. Why not stick the data in a `data-` thing on a `div` or something like that?